demo/python/porcupine_demo_mic.py

```diff
@@ -110,7 +110,25 @@ def run(self):
                 result = porcupine.process(pcm)
                 if result >= 0:
                     print('[%s] Detected %s' % (str(datetime.now()), keywords[result]))
-
+        except pvporcupine.PorcupineInvalidArgumentError as e:
+            print("One or more arguments provided to Porcupine is invalid")
```

Comment:
```suggestion
            print(f"One or more arguments provided to Porcupine is invalid: {args}")
```

Reply:
No `args` object in the mic demo unfortunately, we use a class based approach

Reply:
ah, I see. In that case, ignore this suggestion, but I see that `args.access_key` is used in the following cases, so you should replace those with `'{self._access_key}'`